tests/queries/0_stateless/00002_log_and_exception_messages_formatting.sql

```diff
@@ -58,8 +32,14 @@ select 'messages shorter than 10', max2(countDistinctOrDefault(message_format_st
 -- Same as above. Feel free to update the threshold or remove this query if really necessary
 select 'messages shorter than 16', max2(countDistinctOrDefault(message_format_string), 3) from logs where length(message_format_string) < 16 and message_format_string not in known_short_messages;
 
--- Same as above, but exceptions must be more informative. Feel free to update the threshold or remove this query if really necessary
-select 'exceptions shorter than 30', max2(countDistinctOrDefault(message_format_string), 30) from logs where length(message_format_string) < 30 and message ilike '%DB::Exception%' and message_format_string not in known_short_messages;
+-- Unlike above, here we look at length of the formatted message, not format string. Most short format strings are fine because they end up decorated with context from outer or inner exceptions, e.g.:
```

Comment:
We still should avoid short format strings like `Write file: {}` or `Error: {}`, but the two queries above will check this, so it's fine. And the check for exception messages length looks much better now